extensions/amp-date-display/1.0/test/validator-amp-date-display.out

```diff
@@ -192,6 +192,26 @@ amp-date-display/1.0/test/validator-amp-date-display.html:168:2 The attribute 'd
 amp-date-display/1.0/test/validator-amp-date-display.html:173:2 The attribute 'datetime' in tag 'amp-date-display' is set to the invalid value '2017-08-02T15:05:05.+04:00'. (see https://amp.dev/documentation/components/amp-date-display)
 |      <template type="amp-mustache">{{iso}}</template>
 |    </amp-date-display>
+|
+|    <!-- valid, with localeOptions json -->
+|    <amp-date-display datetime="now" layout="fixed" width="360" height="20">
+|      <script type="application/json">
+|        { "localeOptions": { "timeStyle": "short" } }
+|      </script>
+|      <template type="amp-mustache">{{localeString}}</template>
+|    </amp-date-display>
+|
+|    <!-- invalid: AMP Runtime can't parse json with comments. -->
+|    <amp-date-display datetime="now" layout="fixed" width="360" height="20">
+|      <script type="application/json">
+>>     ^~~~~~~~~
+amp-date-display/1.0/test/validator-amp-date-display.html:187:4 The script tag contains invalid JSON that cannot be parsed.
+|        {
+|          // Comment, should break parsing and result in a warning.
+|        }
+|      </script>
+|      <template type="amp-mustache">{{localeString}}</template>
+|    </amp-date-display>
 |  </body>
 |
-|  </html>
+|  </html>
```

Comment:
Nit: newline

Reply:
Can I know more about the details? 
There was no newline at the end of the file originally (I added lines in between body). 
Also found that there's no new line in other out file (e.g. https://github.com/ampproject/amphtml/blob/main/extensions/amp-timeago/1.0/test/validator-amp-timeago-mutation.out)